documentation/api/development.md

```diff
@@ -25,13 +25,13 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
 
 ## Run the application
 
-1. In your terminal, `cd` to the `api` directory of this repo.
+1. Run `cp frontend/.env.development frontend/.env.local` to create your local copy of the frontend environment variables
```

Comment:
Considering this is specifically in the api development doc, I'm thinking this isn't necessary here. 
As an aside, we seem to have different conventions for the frontend and backend dev setup docs, and there is no reason for that.

Reply:
duly removed!